ferrocene/ci/docker-images/ubuntu-20/Dockerfile

```diff
@@ -149,3 +146,12 @@ RUN mkdir /home/ci \
     && chown -R ci: /home/ci
 USER ci
 WORKDIR /home/ci
+
+# We prefer uv over other solutions, it's in Rust and fast
+ENV PATH="/home/ci/.local/bin:$PATH"
+RUN python3 -m pip install uv
```

Comment:
One thing you can do here is copy a built `uv` from our base image instead of installing it with `pip` or `pipx` (example in https://github.com/astral-sh/uv/issues/4385)

Reply:
I was thinking of installing `uv` from your curl command, but that'd work too I think! I think `uv toolchain` can replace this: https://github.com/ferrocene/ferrocene/blob/40f588b69a7af489c78c438f15e11a5f6c2f412d/ferrocene/ci/docker-images/ubuntu-20/Dockerfile#L74-L85

Reply:
Oh you build from source! Eek. Yes you could use `uv toolchain` to fetch a Python. There are some [behavioral quirks](https://gregoryszorc.com/docs/python-build-standalone/main/quirks.html) to the standalone builds, but usually it's great — I'm not sure about your exact use case.